src/routes/EResourcesRoute.js

```diff
@@ -9,17 +9,17 @@ import { getSASParams } from '@folio/stripes-erm-components';
 import View from '../components/views/EResources';
 import NoPermissions from '../components/NoPermissions';
 import { urls } from '../components/utilities';
+import { resultCount } from '../constants';
 
-const INITIAL_RESULT_COUNT = 100;
-const RESULT_COUNT_INCREMENT = 100;
+const RESULT_COUNT_INCREMENT = resultCount.RESULT_COUNT_INCREMENT;
 
 class EResourcesRoute extends React.Component {
   static manifest = Object.freeze({
     eresources: {
       type: 'okapi',
       path: 'erm/resource/electronic',
       records: 'results',
-      recordsRequired: '%{resultCount}',
+      resultOffset: '%{resultOffset}',
       perRequest: 100,
```

Comment:
Same as the other PR
```suggestion
      perRequest: resultCount.RESULT_COUNT_INCREMENT,
```